openedx/core/djangolib/tests/test_markup.py

```diff
@@ -100,7 +100,7 @@ def test_strip_all_tags_but_br_returns_html(self):
 
         html = strip_all_tags_but_br('{name}<br><script>')
         html = html.format(name='Rock & Roll')
-        self.assertEqual(html.decode(), u'Rock &amp; Roll<br>')
```

Comment:
Why did this work before?  Did we used to get a string but now we're getting a markup object?

Reply:
@mraarif already fixed this, referenced PR: https://github.com/edx/edx-platform/pull/21808